pkg/aws/services/cloudfront/cloudfront.go

```diff
@@ -127,6 +127,7 @@ func (s *Service) EnsureDistribution(config DistributionConfig) (*Distribution,
 
 	// Add internal and customer tags.
 	{
+		customerTags := config.CustomerTags
```

Comment:
```suggestion
		customerTags := config.CustomerTags.DeepCopy()
```

so we don't mutate the input parameter which may lead to surprises

Reply:
DeepCopy function does not exists for map[], so doing the ugly copy via `for`